src/main/java/org/sagebionetworks/web/client/widget/ReactComponentDiv.java

```diff
@@ -1,20 +1,56 @@
 package org.sagebionetworks.web.client.widget;
 
-import org.gwtbootstrap3.client.ui.html.Div;
+import org.sagebionetworks.web.client.jsinterop.ReactDOMRoot;
+import org.sagebionetworks.web.client.jsinterop.ReactNode;
+
+import com.google.gwt.user.client.ui.HTMLPanel;
 
 /**
  * Automatically unmounts the ReactComponent (if any) inside this div when this container is detached/unloaded.
  */
-public class ReactComponentDiv extends Div {
+public class ReactComponentDiv extends HTMLPanel {
+
+	private ReactDOMRoot root;
+	private ReactNode component;
```

Comment:
Need to store the component because sometimes this Widget could unload and load without being re-configured